career-paths/IoT-dev/contributors.md

```diff
@@ -0,0 +1 @@
+Taimoor khan
```

Comment:
Do we need this file? @Rubix982 @azmaktr

Reply:
Currently we haven't had a discussion if we wanted to make a contributors file, so I'm unsure.
We can put this idea in the backlog - however, if we want to pursue with this idea. But for now, there has been no talk about this.